README.md

```diff
@@ -47,23 +48,38 @@
 * [SendCrypto Bot](https://sendcryptobot.io/) - Tip with Dai on Twitter, Slack and Reddit.
 * [Unicef Ventures, France](https://lp.unicef.fr/donate-dai/) - Invests in frontier technologies, and how they could be used to solve humanity’s greatest challenges.
 
+
 ## E-commerce
 
 * [Bidali](https://giftcards.bidali.com/buy-giftcards-with-dai/) - Buy gift cards with Dai.
+* [OpenSea](https://opensea.io) - Buy and sell digital good and assets.  Make offers in Dai.
+
+## Payments
+
+* [Aave Pay](https://pay.aave.com/) - Use Dai and other crypto to send fiat payments to basically anyone.
+* [Coingate](https://coingate.com) - Accept payment in Dai and 50 other crypto currencies.
+* [Dai Card](https://daicard.io/) - Instant, off-chain, peer to peer, Dai transactions using browser-based UI.
+* [Wirex](https://wirexapp.com/card/) - Fund a Wirex debit card with Dai and spend anywhere Visa is accepted.
```

Comment:
Couldn't find any mention of Dai on their site. Where did it say they allow you to use it?

Reply:
https://wirexapp.com/blog/post/a-token-to-dai-for-0004 According to this blog entry, they do support it. I found mentions on Reddit of people using Dai with a wirex cars too.